posix/clone

```diff
@@ -103,3 +103,15 @@ tag)
 	clone-commit
 	;;
 esac
+
+if [[ ! -z "$DRONE_OUTPUT_FILE_PATHS_CONTENT" ]]
+then
+  for FILE_PATH in ${DRONE_OUTPUT_FILE_PATHS_CONTENT//,/ }
+  do
+      # call your procedure/other scripts here below
+      FILE_CONTENT="$(<"$FILE_PATH")"
+      FILE_CONTENT_BASE_64=`echo $FILE_CONTENT | base64`
+      printf "writing drone outputs with file path content %s=%s" $FILE_PATH $FILE_CONTENT_BASE_64
```

Comment:
printf works best with bash but this script runs with bin/sh https://stackoverflow.com/a/25894347

Reply:
can we remove the printf statement

Reply:
I will address this comment @shubham149